### DISCUSSION TITLE
Support on linux distributions

### DISCUSSION THREAD

**Discussion by horvatm** [biolab/orange3#5488] [General]
Hi.

On which Linux distributions Orange was tested or is continually tested, namely I am running into issues with QT on Ubuntu 20.04.2. This seems be a known very old issue
```
    from PyQt5.QtWebKitWidgets import *
ValueError: PyCapsule_GetPointer called with incorrect name
```

Best regards,

Martin

**Comment by markotoplak** [comment]
Hm, I am running it on the same Ubuntu perfectly. How did you install Orange? For mine, I set up a python virtualenv and installed it there (with pip). Works like a charm, I even tried it anew a few days ago.

What does your `pip freeze | grep -i pyqt` say? Mine, in my Orange virtualenv, is

```
PyQt5==5.12.3
PyQt5-Qt==5.15.2
PyQt5-sip==12.8.1
-e git+git@github.com:pyqtgraph/pyqtgraph.git@cd40562121b843a0da76bc407283f9967d4c2c4a#egg=pyqtgraph
PyQtWebEngine==5.12
```

**Reply by markotoplak** [reply]
Also, I see that at least on 18.04 you had to do `sudo apt-get install -y libxkbcommon-x11-0` for the pyqt to work.

**Reply by horvatm** [reply]
Thanks for the fast reply. I have created a fresh user and locally installed orange
```
pip3 install orange3 --user | grep Qt | sed -e 's/^.*: //g'
```
and it gave me
```
PyQt5!=5.15.1,>=5.12 in ./.local/lib/python3.8/site-packages (from orange3) (5.15.4)
PyQtWebEngine>=5.12 in ./.local/lib/python3.8/site-packages (from orange3) (5.15.4)
AnyQt>=0.0.11 in ./.local/lib/python3.8/site-packages (from orange3) (0.0.13)
PyQt5-Qt5>=5.15 in ./.local/lib/python3.8/site-packages (from PyQt5!=5.15.1,>=5.12->orange3) (5.15.2)
PyQt5-sip<13,>=12.8 in ./.local/lib/python3.8/site-packages (from PyQt5!=5.15.1,>=5.12->orange3) (12.9.0)
PyQtWebEngine-Qt5>=5.15 in ./.local/lib/python3.8/site-packages (from PyQtWebEngine>=5.12->orange3) (5.15.2)
```

**Reply by markotoplak** [reply]
Doing `--user` installs can be problematic, because some packages from you user folders and your Ubuntu installation may clash. Instead, try this:

```
virtualenv -p python38 orangetest
source orangetest/bin/activate
pip install orange3
```

Whenever you will need to start Orange, you will need to activate the virtual environment:

```
source orangetest/bin/activate
orange-canvas
```
